### PR TITLE
[G2M] README - reorg and highlight sections, include ws directly

### PR DESCRIPTION
**Pull request by woozyking**
quick link to rendered MD: https://github.com/EQWorks/eqworks-intro/blob/restructure/README.md

also relevant to the pending issue raised by @YizhiCatherineZhang to highlight the hiring process so our job applicants can find it easily

### CODE REVIEW

**Review by ultraviollette — APPROVED** [review]

**Review by amy83762100 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
+
+## 💻 Work Samples
+
+At EQ Works, we take a more effective path to the technical interview. Instead of solving abstract algorithms and data structure problems, you'll get to work on an open-book and open-ended work sample that resembles our actual work. If you are a maker, you may find the problems easier (perhaps more enjoyable) yet much more meaningful to understand how well you can integrate into our team from a technical perspective. We encourage you to check out [our hiring philosophy](https://github.com/EQWorks/common/blob/main/hiring.md) to get more insights.
+
+## 🙌 Open Presence
+
+We have a [LOCUS Engineering Blog](https://medium.com/locus-engineering) maintained by our product/engineering teams. In it, we share our thoughts on our product and technology, culture, and more.
+
+You can also find us on GitHub with a growing portfolio of [open-source projects](https://github.com/EQWorks?q=&type=public), made with our humble effort sincere hope to give back to the community.
```
Comment (kevlabs) [inline README.md]: humble effort **and** sincere hope?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+
+## 👥 Join us!
+
+If you are interested in working with us but haven't applied yet, please go to https://apply.workable.com/eqworks/ and check out the roles.
```
Comment (kevlabs) [inline README.md]: Maybe add hyperlink here.

**Review by kevlabs — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (the-learning-mechanic) [reply README.md]: You can find our growing portfolio of [open-source projects](https://github.com/EQWorks?q=&type=public) on GitHub; in the spirit of open contribution to the wider geospatial analytics and machine learning community.

**Review by the-learning-mechanic — CHANGES_REQUESTED** [review]
You can find our growing portfolio of open-source projects on GitHub; in the spirit of open contribution to the wider geospatial analytics and machine learning community.

File: README.md (hunk 2 above)
Comment (woozyking) [reply README.md]: github auto renders the hyperlink, but i'll make it `<link>` anyway for portability

File: README.md (hunk 1 above)
Comment (woozyking) [reply README.md]: good suggestions, will incorp

though i won't be that specific about which particular community

**Review by RamenLover88 — APPROVED** [review]

**Review by the-learning-mechanic — APPROVED** [review]

**Review by ultraviollette — APPROVED** [review]

**Pull request merged**